src/integrationTest/resources/application.yaml

```diff
@@ -10,6 +10,8 @@ uk:
           applicant2SignInDivorceUrl: ${NOTIFY_TEMPLATE_APPLICANT2_SIGN_IN_DIVORCE_URL:https://nfdiv-apply-for-divorce.aat.platform.hmcts.net/applicant2}
           signInDissolutionUrl: ${NOTIFY_TEMPLATE_SIGN_IN_DISSOLUTION_URL:https://nfdiv-end-civil-partnership.aat.platform.hmcts.net/}
           applicant2SignInDissolutionUrl: ${NOTIFY_TEMPLATE_APPLICANT2_SIGN_IN_DISSOLUTION_URL:https://nfdiv-end-civil-partnership.aat.platform.hmcts.net/applicant2}
+          switchToSoleApplicationDivorceUrl: ${NOTIFY_TEMPLATE_SWITCH_TO_SOLE_APPLICATION_DIVORCE_URL:https://nfdiv-apply-for-divorce.aat.platform.hmcts.net/switch-to-sole-application}
```

Comment:
also please set them in values.yaml as we have done for `NOTIFY_TEMPLATE_SIGN_IN_DIVORCE_URL: https://nfdiv-apply-for-divorce.{{ .Values.global.environment }}.platform.hmcts.net/` so that it is correctly set for each environment.